reference/types/array.md

```diff
@@ -1,5 +1,36 @@
 # Array
 
+## The Concept
+
 A [collection][type-collection] data structure often consisting of a of elements of the same type, each identified by an array index. The simplest type of array is a linear (one-dimensional) array.
 
+## What to cover
+
+Tell a student what an array is in your language, how an array can be manipulated, and ensure they understand where to look for docs on arrays.
+
+- **Explain what arrays are in your language?** Are the arrays fixed sized or not? Can an array contain elements of different types?
+- **How to access an array element?** What is the syntax for accessing an array element? Where do indexes start from in your language?
+- **How to create an array?** What is the syntax for creating a new array?
+- **How to add or remove elements from an array?** What is the syntax for various manipulations of an array?
+
+## Exercises
+
+### Stack of cards
+
+This exercise deals with manipulatating a stack of playing cards. The reference implementation (Javascript) teaches:
+
+- Create an array
+- Accessing an element from an array
+- Changing an element from an array
+- Deleting an element from an array
+- Adding an element to an array
+- Using the length of an array
+
+#### Implementations
+
+| Track | Exercise                          | Changes |
+| ----- | --------------------------------- | ------- |
+| JS    | [arrays-basic][implementation-js] | None    |
+
 [type-collection]: ./collection.md
+[implementation-js]: ../../languages/javascript/exercises/concept/arrays-basic/.docs/introduction.md
```

Comment:
This is the dead link.

Reply:
How is this a dead link? Aren't the following also considered dead links (inside [this file](https://github.com/exercism/v3/blob/master/reference/types/enum.md))?

![image](https://user-images.githubusercontent.com/17279716/79262072-65215200-7e99-11ea-9cd3-71d11ec8eb8c.png)

Reply:
It works fine.

Reply:
The `../../` goes from the `languages/javascript/reference/types` directory to the `languages/javascript/` directory. There is no `languages` directory within the `javascript` directory, which it is trying to find.

Reply:
The others are probably also dead links, but the link checker hadn't got up to them before it found this first dead link.

Reply:
The thing is that it does not go from the `languages/javascript/reference/types`, but from the generic, language-agnostic, `reference/types` directory

Reply:
Oh yes I see now. Interestingly, the `languages/javascript` is `languages/JavaScript` in the files changed section of this commit. Perhaps this is why the link checker is failing?

Reply:
This is so strange. On both master and this branch (locally) the directory is all lowercase.

Reply:
There seems to be a `JavaScript` directory on the [master ](https://github.com/exercism/v3/tree/master/languages)branch. This thing is beyond my knowledge.

Reply:
#1236 
Perhaps this PR will fix it? /shrug

Reply:
Case-insensitivity :/